app/models/comment.rb

```diff
@@ -1,4 +1,9 @@
 class Comment < ApplicationRecord
   belongs_to :user
   belongs_to :post
```

Comment:
- [ ] Kindly I Recommend you to specifying that the `author `association should map to instances of the `User class`. This is useful when the name of the association (author) doesn’t match the name of the associated model (User) just like below.

```
belongs_to :author, class_name: 'User'
belongs_to :post

```